contents/tutorials/stripe-reports.md

```diff
@@ -0,0 +1,185 @@
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/link_light_d827d9f83f.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/link_dark_6622bd98c2.png"
+  alt="Linking Stripe Account"
+  classes="rounded"
+/>
+
+Once done, PostHog will automatically pull and format your Stripe data for querying. You can adjust the sync frequency, see the last successful run, and more in [data warehouse settings](https://us.posthog.com/data-warehouse/settings/managed).
+
+## Creating insights for your Stripe report
+
+Now that your Stripe data is synced into PostHog, you can use it to create insights for your report. Each of these requires you to create a [new insight in the product analytics tab](https://us.posthog.com/project/insights/new).
+
+### Customer count
+
+To start, we create a trend of customer count over time. 
+
+On the trends tab, change the aggregation to **Unique users**, and then click the data series, go to the **Data Warehouse** tab, hover over the `stripe_customer` table, and click **Select**. You might want to change the **Distinct ID field** from `id` to `email` as Stripe can give multiple `id` values to the same user email.
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/customer_light_944920946a.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/customer_dark_e18fc2d09c.png"
+  alt="Customer Count Insights"
+  classes="rounded"
+/>
+
+This can also be done for any of the other Stripe data like charges, subscriptions, and invoices. You can also add filters based on property values like `created_at`, `email`, `status`, and more.
+
+### Gross revenue
+
+Next, we can get our gross revenue by doing a similar process and selecting `stripe_charge`. For aggregation, we want **Property value sum** and then choose `amount`. We also want to filter out failed charges by clicking the filter button, selecting the `status` property, and making sure it doesn't equal `failed`.
+
+Finally, to clean up the visualization, we can use formula mode to divide by 100 (the `amount` value is in cents) and click **Options** on the chart to add `$` as a prefix.
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/revenue_light_bf87a357f2.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/revenue_dark_e75e42ffb3.png"
+  alt="Gross Revenue Insights"
+  classes="rounded"
+/>
+
+### Monthly recurring revenue
+
+There are many ways to calculate monthly recurring revenue, but the easiest and most common is multiplying the number of customers by the average revenue per customer per month.
```

Comment:
It would be nice if we calculated MRR the [same way that stripe does](https://support.stripe.com/questions/calculating-monthly-recurring-revenue-(mrr)-in-billing#:~:text=Calculating%20Monthly%20Recurring%20Revenue%20(MRR,Billing%20%3A%20Stripe%3A%20Help%20%26%20Support&text=You%20can%20calculate%20Monthly%20Recurring,counts%20%24100%20towards%20your%20MRR.), since otherwise are numbers will differ whats on the stripe dashboard. 

It may be too complicated to do this though and require some HogQL. Up to you if you want to change it